lib/WindowManager.vala

```diff
@@ -185,5 +196,11 @@ namespace Gala {
          * @param action_key The gsettings key of action. Available keys are stored in ActionKeys
          */
         public abstract void launch_action (string action_key);
+
+        /**
+         * Checks whether the action should currently be prohibited.
+         * @return true if the action should be prohibited, false otherwise
+         */
+        public abstract bool action_filter (GestureAction action);
```

Comment:
I think it makes more sense to name is `filter_action`? action_filter is a noun but method names should be verbs

Reply:
I wanted to stay in line with Metas `keybinding_filter` here :shrug:

Reply:
Anyways, I changed it to filter_action :)